packages/frontend/src/singleMovie/components/VideoHeaderComponent.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import { fontFamily } from "../../utils/fontFamily";
+import { fontSize } from "../../utils/fontSize";
+import { fontWeight } from "../../utils/fontWeight";
+import { image } from "../../utils/image";
+import { Colors } from "../../utils/colors";
+import { breakPoint } from "../../utils/breakPoint";
+
+const Header = styled.header`
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 20px;
+`;
+
+const Title = styled.h1`
+  font-family: ${fontFamily.openSansFont};
+  font-size: ${fontSize.headerMobile};
+  font-weight: ${fontWeight.black};
```

Comment:
font-weight: ${fontWeight.black}; , czemu black?

Reply:
to też nie ja pisałem